sdkhelpers.go

```diff
@@ -12,30 +12,28 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package metric
+package otel
 
 import (
 	"context"
-
-	"go.opentelemetry.io/otel/api/core"
 )
 
-// InstrumentImpl is the implementation-level interface Set/Add/Record
-// individual metrics without precomputed labels.
-type InstrumentImpl interface {
+// Instrument interface for implementations of individual metrics without
+// precomputed labels.
+type Instrument interface {
```

Comment:
This is confusing, what does `otel.Instrument` means? Maybe change to `MetricsInstrument`?